spec/shared_print/phase_3_validator_spec.rb

```diff
@@ -49,6 +49,19 @@
       cluster = Cluster.find_by(ocns: [spc.ocn])
       expect { p3v.require_cluster_ht_items(cluster) }.to_not raise_error
     end
+    it "requires compatible policies" do
+      spc.policies = []
+      expect { p3v.require_compatible_policies(spc) }.to_not raise_error
+      # add non-repro, should still be good
+      spc.policies = ["non-repro"]
+      expect { p3v.require_compatible_policies(spc) }.to_not raise_error
+      # add blo, should still be good
+      spc.policies = ["non-repro", "blo"]
+      expect { p3v.require_compatible_policies(spc) }.to_not raise_error
+      # add digitizeondemand should blow it up
+      spc.policies = ["non-repro", "blo", "digitizeondemand"]
+      expect { p3v.require_compatible_policies(spc) }.to raise_error SharedPrint::Phase3Error
```

Comment:
I wonder if this could or should be somewhat more generic like `SharedPrint::InvalidCommitmentError` -- is there a reason to tie it to Phase 3?

Reply:
Yeah, that could be made more generic.